gdpr/anonymizers/model_anonymizers.py

```diff
@@ -292,36 +299,80 @@ def anonymize_qs(self, qs: QuerySet) -> None:
     def deanonymize_qs(self, qs: QuerySet) -> None:
         raise NotImplementedError()
 
+    def get_anonymizer_or_none(self, name):
+        return self.anonymizers.get(name, None)
+
+    def update_related_anonymizer_fields(self, field_name:str, obj: Model, related_fields:Fields,
+                                         legal_reason: Optional[LegalReason] = None,
+                                         purpose: Optional["AbstractPurpose"] = None,
+                                         anonymization: bool = True):
+        anonymizer = self.get_anonymizer_or_none(field_name)
+        related_attribute = getattr(obj, field_name, None)
+
+        if hasattr(obj.__class__, field_name) and related_attribute is None:
```

Comment:
Why is attribute on instance and on class different thing in this case?

Reply:
because getattr can return none if property returns none. I need to check if property exists